buildchain/buildchain/packaging.py

```diff
@@ -148,9 +148,23 @@ def task__build_repositories() -> Iterator[types.TaskDict]:
 
 
 CALICO_CNI_PLUGIN_VERSION : str = '3.8.0'
+METALK8S_SOSREPORT_VERSION : str = '1.0.0'
```

Comment:
Since it's used only at a single place, I don't think you need the constant: you can directly write `version='1.0.0'` l.159.